petab/parameters.py

```diff
@@ -424,20 +424,26 @@ def append_overrides(overrides):
 def get_priors_from_df(
     parameter_df: pd.DataFrame,
     mode: Literal["initialization", "objective"],
+    parameter_ids: Sequence[str] = None,
 ) -> List[Tuple]:
     """Create list with information about the parameter priors
 
     Arguments:
         parameter_df: PEtab parameter table
         mode: ``'initialization'`` or ``'objective'``
+        parameter_ids: A sequence of parameter IDs for which to sample starting points.
+            For subsetting or reordering the parameters.
+            Defaults to all estimated parameters.
 
     Returns:
         List with prior information.
     """
-
     # get types and parameters of priors from dataframe
     par_to_estimate = parameter_df.loc[parameter_df[ESTIMATE] == 1]
 
+    if parameter_ids:
+        par_to_estimate = par_to_estimate.loc[parameter_ids, :]
```

Comment:
Maybe add an error message for the case when provided parameter_ids list contains an id that is not there.